index.html

```diff
@@ -43,7 +43,6 @@
 <a href="bridge.html">Bridge Spec</a>
 <a href="whalegate.html">Whalegate</a>
 <a href="staycheap.html">Will it stay cheap?</a>
-<a href="l2.html">cheapETH has L2</a>
```

Comment:
Why have you removed this link?

Reply:
hi, geohot did here: https://github.com/cheapETH/cheapeth-website/commit/1861375d18be49f95914d6ec660d9e9dd504b69c 

I made my branch from the master of cheaeth/cheapeth-website, maybe that's why. Donno if it should be on homepage or not.

Reply:
I can redo the PRs to CheapEthereum:master to get rid of the conflicts if it helps you. Ping me

Reply:
I just realized something. I'll close this PR and open a new one at cheapetereum.github.io:master. Sorry for the inconvinience, I thought this was just a clone of the old site.

Be back shortly